packages/eslint-mdx/src/traverse.ts

```diff
@@ -17,14 +17,14 @@ export class Traverse {
 
     const children = (node as Parent).children
 
+    this._enter(node, parent)
+
```

Comment:
Maybe your other code did depend on this. Maybe not.

If it did, you might have to make this configurable. But if it works, then your new code now matches NLR depth-first tree traversal, and that‘s used most often!  :+1: https://github.com/syntax-tree/unist#tree-traversal

Reply:
The codes inside does not depend on the order except the comments we just knew. 🤣